Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANS/SANSTubeCalibration.py

```diff
@@ -321,15 +320,15 @@ def PyExec(self):
         self._log_tube_calibration_issues()
         self._notify_tube_cvalue_status(cvalues)
 
-    def _match_workspaces_to_strip_positions(self, ws_list):
+    def _match_workspaces_to_strip_positions(self, ws_list) -> dict[int, any]:
```

Comment:
If you don't want to import `MatrixWorkspace`, perhaps for readability it would be good to use a [stringized type hint](https://github.com/mantidproject/mantid/blob/12e3dd109cc3d771a0cbedb4c562ab60f24101b5/scripts/abins/abinsdata.py#L35)?

Regardless, I think this should be `Any`, from `from typing import Any`. I think `any` here might be the built-in function: https://docs.python.org/3/library/functions.html#any

Reply:
Ah yes, thanks, I thought the `any` didn't seem quite right. I recall trying the stringized type hint at the time but can't recall now why I didn't go for it, will take another look.